buildspec.yml

```diff
@@ -3,20 +3,19 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      python: 3.10    
+      python: 3.10
     commands:
       - pip install --upgrade pip
+      - pip install -r source/lambda/requirements.txt -t source/lambda/
   pre_build:
     commands:
       - echo "Installing dependencies and executing unit tests - `pwd`"
-      - cd source
-      - chmod +x ./run-all-tests.sh && ./run-all-tests.sh
+      - cd deployment
+      - chmod +x ./run-unit-tests.sh && ./run-unit-tests.sh
```

Comment:
on the pipeline yes, but run locally this can sometimes fail so this just ensures that the script doesn't get tripped up by a stupid permissions issue. (this script will likely die completely in the coming updates as well)

Reply:
I'm inclined to believe that if a user unsets the executable bit on a file cloned from a repo, or sets their `umask` so that happens, they don't want that file to execute.